docs_src/microservices/security/Ch-APIGateway.md

```diff
@@ -128,6 +128,40 @@ will be denied when trying to access resources through the API Gateway.
     name = "acl"
     whitelist = "admin,user"
 
+**Configuration of Adding Microservices Routes for API Gateway**
+
+For the current pre-existing Kong routes, they are configured and initialized statically through configuration TOML file specified in `security-proxy-setup` application. This is not sufficient for some other new additional microservices like application services, for example.  Thus, adding new proxy Kong routes are now possibly achieved via environment variable, `ADD_PROXY_ROUTE`, of service `edgex-proxy` in the docker-compose file.  Here is an example:
+
+```yaml
+edgex-proxy:
+      ...
+    environment:
+      ...
+      ADD_PROXY_ROUTE: "myApp.http://my-app:56789"
+      ...
+
+...
+
+myApp:
+   ...
+   container_name: myApp
+   hostname: myApp
+   ...
+  
+```
+
+The value of `ADD_PROXY_ROUTE` takes a comma-separated list of one or more (at least one) paired additional service name and URL for which to create proxy Kong routes.   The paired spec. is given like the following:
+
+**\<ServiceName\>.\<RouteURL\>**
+
+where **ServiceName** is the name of service which requests to create proxy Kong route; it is the docker network hostname of the service wanted to add the proxy Kong route in the docker-compose file if running from docker-compose, for example, `myApp` in this case.
+
+**RouteURL** is the full qualified URL for the service, like `http://my-app:56789`
+
+So as an example, for a single service, the value of `ADD_PROXY_ROUTE` would be:  "`my-app.http://my-app:56789`".
+
+Once `ADD_PROXY_ROUTE` is configured and composed-up successfully, the porxy route then can be accessed the app's REST API via Kong as `http://localhost:8000/myApp/api/v1/...` in the same way you would access the edgex service in which you will also need an acess token as well.
```

Comment:
s/porxy/proxy

Reply:
Are these routes added to the "admin" group/role?  If so, specify that as well.  (i.e. tell users what kind of kong token they need to create to get access to the newly added routes).

Reply:
they are treated as the default ACL role because the ACL configuration of these are pre-defined in the configuration toml: see https://github.com/edgexfoundry/edgex-go/blob/master/internal/security/proxy/service.go#L184